web/themes/custom/cj_material/sass/styles.scss

```diff
@@ -1378,276 +1371,128 @@ article footer {
 
 /* BP - Job Post Page */
 
-.form-item label {
-  font-size: 20px;
-  font-weight: bold;
-  font-family: Montserrat;
-  line-height: 2em;
-}
+.form-item {
+  label, h4 {
+    font-size: 20px;
+    font-weight: bold;
+    font-family: Montserrat;
+    line-height: 2em;
+  }
+  &:focus-within label {
+    color: #FF904B;
+  }
 
-.form-item input[type=text], .form-item input[type=email], .form-item input[type=password], .form-item input[type=search], .form-item input[type=tel], .form-item textarea, .form-item select {
+  input[type=text], input[type=url], input[type=date], input[type=number], input[type=email], input[type=password], input[type=search], input[type=tel], textarea, select {
     display: block;
     width: 100%;
     max-width: 1200px;
     border: 1px solid rgba(0, 0, 0, 0.12);
     padding: 8px 0;
+    text-indent: 10px;
+    background-color: #F6F4F4;
+
+    &:focus {
+      border: 2px solid #FF904B;
+      padding-bottom: 7px;
+    }
+    &[type=date], &[type=number] {
+      width: initial;
+      display: inline-block;
+    }
+
+  }
+}
+.form-wrapper {
+  h4 {
+    font-size: 20px;
+    font-weight: bold;
+    font-family: Montserrat;
+    line-height: 2em;
+    color: rgba(0,0,0,0.54);
+  }
 }
 
-input#edit-label-0-value {
-    background-color: #f6f4f4;
+.field--type-text, .field--type-string, .field--type-datetime, .field--type-text-long {
+  input, textarea {
+    background-color: #F6F4F4;
     text-indent: 10px;
+  }
 }
 
-textarea#edit-description-summary-0-value {
-    background-color: #f6f4f4;
-    text-indent: 10px;
+.proximity-filter-container .form-item input[type=text] {
+  display: inline-block;
 }
 
-form#job-role-post-form {
-    padding-left: 20px;
-    padding-right: 20px;
+form {
+  padding-left: 20px;
+  padding-right: 20px;
 }
 
-/* BP Post a Job 21/1/2020 */
+/* BP Changes to Post A Job Page */
 
 summary {
```

Comment:
Lets move this chunk up near the top of the file with the other details settings.

Reply:
Done.